docs/docs/how_to/pydantic_compatibility.md

```diff
@@ -1,27 +1,102 @@
 # How to use LangChain with different Pydantic versions
 
-- Pydantic v2 was released in June, 2023 (https://docs.pydantic.dev/2.0/blog/pydantic-v2-final/)
-- v2 contains has a number of breaking changes (https://docs.pydantic.dev/2.0/migration/)
-- Pydantic v2 and v1 are under the same package name, so both versions cannot be installed at the same time
+- Pydantic v2 was released in June, 2023 (https://docs.pydantic.dev/2.0/blog/pydantic-v2-final/).
+- v2 contains has a number of breaking changes (https://docs.pydantic.dev/2.0/migration/).
 
-## LangChain Pydantic migration plan
+As of `langchain>=0.0.267`, LangChain allows users to install either Pydantic V1 or V2.
 
-As of `langchain>=0.0.267`, LangChain will allow users to install either Pydantic V1 or V2. 
-   * Internally LangChain will continue to [use V1](https://docs.pydantic.dev/latest/migration/#continue-using-pydantic-v1-features).
-   * During this time, users can pin their pydantic version to v1 to avoid breaking changes, or start a partial
-   migration using pydantic v2 throughout their code, but avoiding mixing v1 and v2 code for LangChain (see below).
+Internally, LangChain continues to use the [Pydantic V1](https://docs.pydantic.dev/latest/migration/#continue-using-pydantic-v1-features) via
+the v1 namespace of Pydantic 2.
 
-User can either pin to pydantic v1, and upgrade their code in one go once LangChain has migrated to v2 internally, or they can start a partial migration to v2, but must avoid mixing v1 and v2 code for LangChain.
+Because Pydantic does not support mixing .v1 and .v2 objects, users should be aware of a number of issues
+when using LangChain with Pydantic.
+
+:::note
+
+Pydantic 1 End of Life was in June 2024. LangChain will be dropping support for Pydantic 1 in the near future, 
```

Comment:
shoudl this be one of the bullets at the top?

also would explicitly say something to effect of "This change will be accompanied with a minor version bump across all relevant LangChain libraries"